test/smoke.bats

```diff
@@ -140,3 +140,21 @@ load 'helper'
   assert_json_eq '[.events[] | select(.http_client_request)] | length' 3
   assert_json_eq '[.events[] | select(.http_client_response)] | length' 3
 }
+
+# bats captures stdout and stderr to the same variable ($output). We
+# need to hide the informational message from the agent commands so
+# json assertions don't get confused.
+@test "appmap agent init" {
+  run bash -c 'java -jar /appmap.jar -d /spring-petclinic init 2>/dev/null'
+  assert_success
+
+  assert_json_contains '.configuration.contents' 'path: org.springframework.samples.petclinic'
```

Comment:
@kgilpin it doesn't seem like the `init` command is handling the `-d` flag correctly. This test is failing because `init` isn't find the petclinic source.